Package.swift

```diff
@@ -301,10 +310,17 @@ let package = Package(
       ],
```

Comment:
Should a define be added above for `Crashlytics watchOS SDK`?

Reply:
Yep. Good catch!